src/Guide/Api/Types.hs

```diff
@@ -158,6 +180,27 @@ data SearchSite route = SearchSite
 
 type Api = ToServant Site AsApi
 
+-- | Trait type (Pro/Con) and instancies.
+data TraitType = Pro | Con
+    deriving (Show, Generic)
+
+instance ToSchema TraitType where
+    declareNamedSchema = genericDeclareNamedSchema schemaOptions
+
+instance ToParamSchema TraitType where
+    toParamSchema _ = mempty
+        & S.type_ .~ SwaggerString
+        & S.format ?~ "Trait type"
+
+instance ToHttpApiData TraitType where
+    toUrlPiece = toText . show
```

Comment:
Can you make it lowercase for consistency? `/item/:id/trait/Pro` works, of course, but looks weird.